_includes/head.html

```diff
@@ -1,13 +1,48 @@
+{% if page.title %}
+  {% assign title = page.title %}
+{% else %}
+  {% assign title = site.title %}
+{% endif %}
+
+{% if page.description %}
+  {% assign description = page.description | strip_html | strip_newlines | truncate: 160 %}
+{% else %}
+  {% assign description = site.description %}
+{% endif %}
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>{% if page.title %}{{ page.title }} | {% endif %}{{ site.title }}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
```

Comment:
Seems like we're dropping support for `excerpt` (which I think is auto generated for blog posts?) -- is that intentional?

Reply:
Ahh was unaware `excerpt` was autogenerated, didn't find it anywhere in the code base so had assumed it was unused. Have added support for this back in, if the description is not explicitly overridden. 069a8c8

Reply:
Hrm, seems we may need to opt-in to them being generated: https://jekyllrb.com/docs/pages/#excerpts-for-pages. Feels like it would be a good thing to opt into though!

Reply:
👍 d13ee2e